Lib/test/test_difflib.py

```diff
@@ -501,12 +501,30 @@ def test_is_character_junk_false(self):
         for char in ['a', '#', '\n', '\f', '\r', '\v']:
             self.assertFalse(difflib.IS_CHARACTER_JUNK(char), repr(char))
 
+class TestFindLongest(unittest.TestCase):
+    def test_default_args(self):
+        sm = difflib.SequenceMatcher(a='foo bar', b='foo baz')
+        match = sm.find_longest_match()
+        self.assertEqual(match.a, 0)
+        self.assertEqual(match.b, 0)
+        self.assertEqual(match.size, 6)
```

Comment:
Would be good to also test a mix of explicit and default args.  For example, like this test, but with `alo=2` and `bhi=6`.